### PR TITLE
Medibot recipe now requires medkit to be empty

### PR DESCRIPTION
**Pull request by GoodWheatley**
## About the PR 

Took a stab at fixing #10852 and added the ContainerEmpty condition to the medibot recipe.

**Changelog**

:cl:
- fix: You can no longer delete items by crafting medibots

### CODE REVIEW

File: Resources/Prototypes/Recipes/Crafting/Graphs/bots/medibot.yml [hunk 1]
```diff
@@ -28,6 +31,6 @@
           sprite: Mobs/Silicon/drone.rsi
           state: l_hand
         name: borg arm
-        doAfter: 2
+        doAfter: 2
```
Comment (0x6273) [inline Resources/Prototypes/Recipes/Crafting/Graphs/bots/medibot.yml:34]:
```suggestion
        doAfter: 2
```

**Comment by GoodWheatley** [comment]
The tests keep failing :(

**Review by Just-a-Unity-Dev — CHANGES_REQUESTED** [review]
You have an extra space

**Comment by Just-a-Unity-Dev** [comment]
Ignore my previous comment

**Comment by GoodWheatley** [comment]
> Ignore my previous comment

no problem

**Comment by Just-a-Unity-Dev** [comment]
GitHub doublesended my comment, great

**Comment by GoodWheatley** [comment]
wait that wasn't supposed to be pushed fuck

**Comment by rolfero** [comment]
Your tabs seem to be off - its important to have the correct amount of tabs to "place" the condition correctly. It should be under the specific edge. Perhaps double check https://docs.spacestation14.io/en/content/construction/index and see if you can get it working.

**Comment by Mervill** [comment]
This does not fix the problem, for reasons I discussed in the original issue. Testing this change using a standard filled medkit (`MedkitFilled`) still creates the medibot.

**Comment by GoodWheatley** [comment]
> This does not fix the problem, for reasons I discussed in the original issue. Testing this change using a standard filled medkit (`MedkitFilled`) still creates the medibot.

I would fix the underlying problem if I could, but I still have no idea how to do it. If somebody makes a PR that fixes the underlying issue I'll close this.

File: Resources/Prototypes/Recipes/Crafting/Graphs/bots/medibot.yml [hunk 2]
```diff
@@ -29,5 +29,8 @@
           state: l_hand
         name: borg arm
         doAfter: 2
+    conditions:
+      - !type:ContainerEmpty 
+        container: "medkit"
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Recipes/Crafting/Graphs/bots/medibot.yml:34]: pretty sure this needs to be `storagebase`.

**Comment by GoodWheatley** [comment]
@Zumorica ready for review

**Review by vulppine — DISMISSED** [review]

**Review by Mervill — CHANGES_REQUESTED** [review]
This condition is an `IGraphCondition` which is not evaluated when 'crafting' something (vs constructing something which does check `IGraphCondition`). So this change does not fix the issue.

Unfortunately this problem can't be solved with a simple data change. What really needs to happen is crafting needs to be rewritten to not use graphs (#11374). Which is a much bigger job.

**Comment by GoodWheatley** [comment]
In light of Mervill's comment, going to close this since it ultimately won't solve the problem.

**Comment by veliebm** [comment]
Good try @GoodWheatley :)

**Pull request closed without merging**